app/src/main/java/com/sun/chat_04/ui/friend/search/SearchConstract.kt

```diff
@@ -0,0 +1,16 @@
+package com.sun.chat_04.ui.friend.search
+
+import com.sun.chat_04.data.model.Friend
+import com.sun.chat_04.ui.BasePresenter
+import com.sun.chat_04.ui.BaseView
+
+interface SearchConstract {
+    interface View : BaseView {
+        fun onGetListUserSuccesfully(friends: ArrayList<Friend>)
```

Comment:
```suggestion
        fun onGetUsersSuccesfully(friends: ArrayList<Friend>)
```
nên sử dụng danh từ số nhiều,
tương tự cho failure